validator.rb

```diff
@@ -0,0 +1,17 @@
+module Validation
+    def self.validateInput(items)
```

Comment:
Always use `snake_casing` for method defenition

Reply:
I have changed it to validate_input. Sharing the commit link below:

https://github.com/chandnissirohi/Ruby-grocery-store-2/commit/ac75fb6f2fab113246e2e5eda217f748daacca93